user-interface/frontend/themes/datamanager/components/all.css

```diff
@@ -0,0 +1,209 @@
+.flex-horizontal {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
I'm unclear on the purpose of the all.css class? 
I guess that we're moving into having all css into a singular file as @KochTobi suggested or is it css classes that are used in general components? Maybe a small comment would help her?